python/py-frozendict/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-frozendict
+version             2.3.8
+revision            0
+categories-append   devel
+license             LGPL-3
+supported_archs     noarch
+platforms           {darwin any}
+
+maintainers         nomaintainer
+
+description         frozendict is a simple immutable dictionary
+
+long_description    frozendict is a simple immutable dictionary. \
+                    It's fast as dict, and sometimes faster! \
+                    Unlike other similar implementations, \
+                    immutability is guaranteed: you can't change \
+                    the internal variables of the class, and they \
+                    are all immutable objects.
+
+homepage            https://github.com/Marco-Sulla/python-frozendict
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}/
+distname            ${python.rootname}-${version}
```

Comment:
distname and master_sites are set by the python PortGroup so these two lines can be removed.

Reply:
I have removed that.